src/components/Inputs/ActionButton/ActionButton.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { ActionButton as UiFabActionButton } from 'office-ui-fabric-react/lib/Button';
+import { styles } from './ActionButton.styles';
+import { classNamesFunction, customizable, styled } from 'office-ui-fabric-react/lib/Utilities';
+
+export function ActionButton(props) {
+	const { label, iconProps, onClick, className, theme, styles } = props;
```

Comment:
Replace label prop with children (```<ActionButton>ButtonLabel</ActionButton>```) to add a possibility to use react-intl's ```FormattedMessage``` as a label. Button and Label components will be updated in scope of Login component development.